app/models/shared_with.rb

```diff
@@ -1,8 +1,9 @@
 class SharedWith
   NAME_VALUES = HashWithIndifferentAccess.new(
-    team:     'Just the team',
-    internal: 'Other teams internally',
-    external:  'Other teams externally'
+    anonymised: 'All identifiable information will be anonymised as we process it',
+    team:       'Identifiable information will only be shared with the research team at Barnardo\'s',
```

Comment:
You can avoid the need to escape the apostrophe by using double quotes on this string and others like it. If you feel like it.